HISTORY.md

```diff
@@ -8,9 +8,13 @@ unreleased
   * deps: debug@2.6.1
     - Allow colors in workers
     - Deprecated `DEBUG_FD` environment variable set to `3` or higher
+    - Fix DEBUG_MAX_ARRAY_LENGTH
+    - Fix Electron reference to `process.env.DEBUG`
     - Fix error when running under React Native
+    - Fix Inline extend function in node implementation
+    - Fix null reference check on window.documentElement.style.WebkitAppearance
```

Comment:
Will our users actually experience this change? Looks like it only applies to web browsers, which this module doesn't function in.